src/log-filestore.c

```diff
@@ -336,8 +348,19 @@ static void LogFilestoreLogCloseMetaFile(const File *ff)
         fprintf(fp, "SIZE:              %"PRIu64"\n", FileTrackedSize(ff));
 
         fclose(fp);
+        /* Move working files to their final location now that we are done
+         * writing them.*/
+        if (rename(working_filename, final_filename) != 0) {
+            SCLogInfo("renaming file %s to %s failed", working_filename,
```

Comment:
these should probably be warnings? OTOH we generally don't want to create output at pkt runtime for the risk of flooding. That risk should be quite low here though.